docs/tokens/lockup.md

```diff
@@ -0,0 +1,118 @@
+---
+id: lockup
+title: Lockups Explained
+sidebar_label: Lockups Explained
+---
+
+These docs include information about lockups in general, how they are implemented on NEAR, some of the challenges this causes and how you can delegate your locked tokens.
+
+
+## Lockup Basics
+
+A "lockup" is when tokens are prevented from being transferred. The configuration of this lockup may vary significantly from case to case but the same smart contract is used for each of them. Accounts which are subject to a lockup have a different setup than accounts which are created without a lockup. If you have a locked-up account (which all accounts will have during the network launch phase), it may be supported slightly differently by various tools (from wallets to delegation interfaces) because of this difference in the architecture.  For example, one wallet has had difficulty displaying correct balances for the accounts because of this.
+
+The most common configuration of lockup on NEAR, particularly for early holders, is to linearly release the tokens for transfer during the entire term of the lockup. For example, a 24 month linear lockup would make a small amount of tokens eligible for transfer with each block that passes until the full amount is free to transfer at the end of 24 months.
+
+Another factor in lockups is the "cliff", which means that no tokens are unlocked until that date (often 12 months after the lockup start).  On that date, a large chunk of tokens is unlocked at once to make it as if the cliff never existed at all.  Most early accounts are subject to a cliff. For example, a 4 year linear lockup with a 1 year cliff will have the following characteristics:
+
+1. Months 0-12: all tokens are locked
+2. Month 12+1 block: the first 25% of the tokens are immediately unlocked
+3. Months 13-48: the remaining 75% of tokens are unlocked smoothly over each block of the remaining 36 months.
+4. Months 48+: all tokens are unlocked
+
+The "start date" of lockups is the date that transfers are enabled on the network ("Phase II") for any contracts distributed prior to Phase II.  For any lockup contracts implemented after this date, it will be simply the date the lockup contract was implemented.
```

Comment:
this is not always the case. Some have an absolute start date that comes before the start of phase 2.